kzz7yh-f53797/cod-ve09v2_kzz7yh-f53797.xml

```diff
@@ -402,8 +404,8 @@
     </p>
                <p xml:id="kzz7yh-f53797-d1e760">
                   <lb ed="#V" n="97"/>Ad primum in oppositum cum dicitur: quod illud mediam 
```

Comment:
I'm deleting "mediam" here because this is actually the first part of "median-te" and i've added "medan" to the line below with "te"

```suggestion
                  <lb ed="#V" n="97"/>Ad primum in oppositum cum dicitur: quod illud 
```